webbanhang.html

```diff
@@ -0,0 +1,470 @@
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <hr>
+            <div class="row">
+                <div class="col-sm-3">
+                    <h5>THÔNG TIN</h5>
+                    <p><a href="#">Giới thiệu</a></p>
+                    <p><a href="#">Sản phẩm</a></p>
+                    <p><a href="#">Tin tức</a></p>
+                    <p><a href="#">Tuyển dụng</a></p>
+                    <p><a href="#">Liên hệ</a></p>
+                </div>
+                <div class="col-sm-3">
+                    <h3>Column 2</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing
+                        elit...</p>
+                    <p>Ut enim ad minim veniam, quis nostrud exercitation
+                        ullamco laboris...</p>
+                </div>
+                <div class="col-sm-3">
+                    <h3>Column 3</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing
+                        elit...</p>
+                    <p>Ut enim ad minim veniam, quis nostrud exercitation
+                        ullamco laboris...</p>
+                </div>
+                <div class="col-sm-3">
+                    <h3>Column 3</h3>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing
+                        elit...</p>
+                    <p>Ut enim ad minim veniam, quis nostrud exercitation
+                        ullamco laboris...</p>
+                </div>
+            </div>
+        </div>
+
+        <script>
+        $(document).ready(function () {
+            // SEARCH
+            $("#search").keyup(function () {
+                // search1($(this));
+                var key = $(this).val().toLowerCase();
+
+                $('.product').show().filter(function () {
+                    return $(this).text().toLowerCase().indexOf(key) == -1;
+                }).hide();
+            });
+
+            
+            // PRODUCT ARRANGEMENT
+            $('.form-control').on('change',function() {
+              
+                // PUT MONEY DATA IN AN ARRAY
+                var money_arr = [];
+                $('.product').each(function(index) {
+                    var prize = $(this).children('.card').children('.card-body').children('.card-text').text();
+                    var last$_position= prize.lastIndexOf('$');
+                    var money = prize.substring(last$_position+1,prize.length-1);  
+                    $(this).addClass(money);
+                    money_arr.push(Number(money));                
+                });
+                // remove duplicate
+                var uniquePrizes = [];
+                $.each(money_arr, function(i, el){
+                    if($.inArray(el, uniquePrizes) === -1) uniquePrizes.push(el);
+                });
+                // SORT DATA
+                var increase_arr = uniquePrizes.slice().sort((a,b) => a-b);
```

Comment:
e thử viết toàn bộ phần xử lý vào 1 function, truyền đối số 0 thì sort increase, 1 thì decrease xem sao

Reply:
để em viết lại :))